tornado/platform/asyncio.py

```diff
@@ -159,10 +157,13 @@ class AsyncIOMainLoop(BaseAsyncIOLoop):
 
        Now used automatically when appropriate; it is no longer necessary
        to refer to this class directly.
+
+    .. versionchanged:: 5.0
+
+       Closing an `AsyncIOMainLoop` now closes the underlying asyncio loop.
```

Comment:
Should there be a test for this?

Reply:
I think a test that just did `assert ioloop.asyncio_loop.is_closed()` would be kind of pointless. Failure to close an asyncio loop would be detected by the "unclosed event loop" log messages, which is what guided me in making these changes. However, I can't figure out how to catch these "exception in destructor" messages to make the tests fail when no one's paying attention to the output (they often don't come until the program shuts down).

Reply:
You could redirect `sys.stderr` temporarily and assert there's nothing there. That's the only way I know of to catch those messages.